### DISCUSSION TITLE
Share the source

### DISCUSSION THREAD

**Discussion by Pemo11235** [af/hyperpaper-planner#12] [Q&A] [answered]
Hello,
really like this planner ! 

I'm a JS dev too, and I would like to help/edit this planner . 
Do you use jsPdf to do this programmatically?  
Do you want to keep it close-source? 
Just know, before to start to dive in on my own 😄 

Thanks 😄

**Comment by af** [comment, marked as answer]
I'm currently not planning to release the code, at least in the next while.

If you're looking to build your own, I'd recommend checking out these libraries:
https://pdfkit.org/
https://react-pdf.org/

It's a fair bit of work to lay everything out and get the linking working properly, but it's fun once you get going :)

**Reply by ert11er** [reply]
İf you are just using a script to do the pdf files, why are you wanting money out of it?
